api/lib/validation/schemaValidator.js

```diff
@@ -57,7 +57,7 @@ var getDaysInMonthInISOFormat = function() {
 };
 
 var getMetricTypes = function() {
-  var metricTypeEnum = ['memoryused'];
+  var metricTypeEnum = ['memoryutil', 'responsetime', 'throughput'];
```

Comment:
Add the upper limit for the threshold value validation, currently there's no upper limit
https://github.com/cloudfoundry-incubator/app-autoscaler/blob/develop/api/lib/validation/schemaValidator.js#L96